src/app/clusters/level-control/level-control.cpp

```diff
@@ -129,14 +131,23 @@ static void reallyUpdateCoupledColorTemp(EndpointId endpoint);
 #define updateCoupledColorTemp(endpoint)
 #endif // IGNORE_LEVEL_CONTROL_CLUSTER_OPTIONS && EMBER_AF_PLUGIN_COLOR_CONTROL_SERVER_TEMP
 
+void emberAfLevelControlClusterServerTickCallback(intptr_t endpointPtr);
+
+static void timerCallback(System::Layer *, void * callbackContext)
+{
+    DeviceLayer::PlatformMgr().ScheduleWork(emberAfLevelControlClusterServerTickCallback,
```

Comment:
Why do we need the extra trip through the event loop here?  Why can't we just do:
```
emberAfLevelControlClusterServerTickCallback(reinterpret_cast<EndpointId>(callbackContext));
```
?